server/src/main/java/org/elasticsearch/TransportVersions.java

```diff
@@ -138,6 +138,7 @@ static TransportVersion def(int id, String uniqueId) {
     public static final TransportVersion V_8_500_072 = def(8_500_072, "e2df7d80-7b74-4afd-9734-aee0fc256025");
     public static final TransportVersion V_8_500_073 = def(8_500_073, "9128e16a-e4f7-41c4-b04f-842955bfc1b4");
     public static final TransportVersion V_8_500_074 = def(8_500_074, "aab0c31c-62d7-4b95-bb29-0b6f367ece64");
+    public static final TransportVersion V_8_500_075 = def(8_500_075, "bd553a42-d9be-497f-aab2-b6be9e26bb4e");
```

Comment:
Suggestion: given https://github.com/elastic/elasticsearch/pull/99500 is about to be merged, maybe you can "plan ahead" and give this version a meaningful name?

Reply:
I would prefer to keep these two separated - if this is merged first, #99500 should be updated to rename it. If #99500 is merged first, I can update this branch with the new scheme.